scripts/exglobal_stage_ic.sh

```diff
@@ -36,21 +36,30 @@ for MEMDIR in "${MEMDIR_ARRAY[@]}"; do
     for ftype in coupler.res fv_core.res.nc; do
       src="${BASE_CPLIC}/${CPL_ATMIC:-}/${PDY}${cyc}/${MEMDIR}/atmos/${PDY}.${cyc}0000.${ftype}"
       tgt="${COM_ATMOS_RESTART_PREV}/${PDY}.${cyc}0000.${ftype}"
-      ${NCP} "${src}" "${tgt}"
-      rc=$?
+      if [ ! -f ${tgt} ]; then
+        ${NCP} "${src}" "${tgt}"
+        rc=$?
+      else
+        rc=0
+      fi
       ((rc != 0)) && error_message "${src}" "${tgt}" "${rc}"
       err=$((err + rc))
     done
     for ftype in ca_data fv_core.res fv_srf_wnd.res fv_tracer.res phy_data sfc_data; do
       for ((tt = 1; tt <= ntiles; tt++)); do
         src="${BASE_CPLIC}/${CPL_ATMIC:-}/${PDY}${cyc}/${MEMDIR}/atmos/${PDY}.${cyc}0000.${ftype}.tile${tt}.nc"
+       #tgt="${COM_ATMOS_RESTART_PREV}/${PDY}.${cyc}0000.${ftype}.tile${tt}.nc"
         if (( tt > 6 )) ; then
             tgt="${COM_ATMOS_RESTART_PREV}/${PDY}.${cyc}0000.${ftype}.nest0$((tt-5)).tile${tt}.nc"
         else
             tgt="${COM_ATMOS_RESTART_PREV}/${PDY}.${cyc}0000.${ftype}.tile${tt}.nc"
         fi
-        ${NCP} "${src}" "${tgt}"
-        rc=$?
+        if [ ! -f ${tgt} ]; then
+          ${NCP} "${src}" "${tgt}"
+          rc=$?
+        else
+          rc=0
+        fi
```

Comment:
If target is present, it should overwrite.  What happens if the source has changed?